pkg/daemon/ceph/osd/daemon.go

```diff
@@ -161,7 +161,7 @@ func configRawDevice(name string, context *clusterd.Context) (*sys.LocalDisk, er
 }
 
 // Provision provisions an OSD
-func Provision(context *clusterd.Context, agent *OsdAgent, crushLocation, topologyAffinity, deviceFilter, metaDevice string) error {
+func Provision(context *clusterd.Context, agent *OsdAgent, crushLocation, topologyAffinity, deviceFilter, metaDevice, nvmeOfFaultDomain string, transferOSDID int) error {
```

Comment:
This is probably not acceptable when contributing this functionaliy to the upstream. You need to find a better way to implement this by adding the codes to right place (TODO).

Reply:
Okay,